redhat-cop/infra-ansible
# Add initial aws network management

### What does this PR do?
Allows for creation/management of AWS VPCs and subnets

### How should this be tested?
Update sample inventory to use valid aws credentials and the vpc / subnet of your choice. Then run the playbook `networks.yml` underneath tests. You should then be able to run it / rerun it to show that it creates the network.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
